docs/source/v3-package-spec.rst

```diff
@@ -579,6 +581,18 @@ of the following values: ``jsonabi``, ``solidity``, ``vyper``.
   :Key: ``type``
   :Type: String
 
+License: ``license``
+^^^^^^^^^^^^^^^^^^^^
+
+The ``license`` field declares the type of license associated with
+this source file. When defined, this license overrides the
+package-scoped `Meta License`_. If undefined, this file extends the
```

Comment:
The last sentence here is a little awkward.  I think it can just be removed assuming the text for the package-scoped license states that it applies to the source files.